README.md

```diff
@@ -1,10 +1,36 @@
 # WeTransfer-iOS-CI
+[![Build Status](https://travis-ci.org/WeTransfer/WeTransfer-iOS-CI.svg?branch=master)](https://travis-ci.org/WeTransfer/WeTransfer-iOS-CI)
+
 Containing all the shared CI logic for WeTransfer repositories
 
-# Features
-- Integrate [SwiftLint](https://github.com/realm/SwiftLint) to lint source code and tests. We have a different config for tests to allow unwrapping for example
+# Why should I use it?
+What's in it for me? Well, quite a lot! With low effort to add it to your project.
+
+- Integrate [SwiftLint](https://github.com/realm/SwiftLint) to lint source code and tests. We have a different config for tests to allow `unwrapping optionals` for example
 - Integrate [Danger](http://danger.systems/) to automatically improve PR reviews
 
+## Danger features
+Following are a list of features which are posted in a comment on PRs based on the submitted files.
+
+  - Warn for big PRs, containing more than 500 lines of code
+  - Warn for missing PR description
+  - Warn for missing updated tests
+  - Show code coverage of PR related files
+  - Show any failed tests
+  - Show all `warnings` and `errors` in the project
+
+### Custom linting
+These warnings are posted inline inside the PR, helping you to solve them easily.
+
+  - Check for `final class` usage
+  - `override` methods without adding logic
+  -  Suggest `weak` over `unowned`
+  -  Suggest `// MARK:` usage for large files
+
+_A comment example. Note that `WeTransferBot` will be replaced by your own bot. More info can be found here: [Getting started with Danger](http://danger.systems/guides/getting_started.html)._
```

Comment:
What do you mean with ‘a comment example’?

Reply:
A **common** example?

Reply:
Haha, nope! It's a comment in that image and this text is the caption. I've updated it to be more describing.